### ISSUE TITLE
Inject Tota11y into preview pages

### ISSUE THREAD

**Issue by migurski** [chimecms/chime#343]
[Tota11y for accessibility](https://khan.github.io/tota11y/) might help editors find accessibility errors.

**Comment by norrishung** [comment]
I saw this tool the other day and it seems like a really great idea.

Although, I'm not sure the preview pages is the right place for it? I would think this tool would mainly be for developers and designers and not content writers (except for maybe the headings?).

**Comment by migurski** [comment]
Good point. It has a few content-related features, like calling out "click here" links and potential weird language, but maybe it’s more of a thing for the designers.
